locale/bg.po

```diff
@@ -21,7 +22,7 @@ msgstr ""
 #: libraries/lib-audio-devices/AudioIOBase.cpp
 #: libraries/lib-note-track/NoteTrack.cpp
 msgid "Stream is active ... unable to gather information.\n"
-msgstr ""
+msgstr "Потокът е активен ... неуспешно събиране на информация"
```

Comment:
ah, and if a line ends in \n in the original, it must end with \n in the translation

Reply:
Cheers! Must've been sleepy and missed those. Thanks again! Ahah!